docs/docs/guide/introduction.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Introduction
+outline: deep
+---
+
+<script setup>
+import { VPTeamMembers } from 'vitepress/theme'
+import { COMMUNITY_TEAM_MEMBERS, CORE_TEAM_MEMBERS } from '../../../.vitepress/theme/constants/team'
+</script>
+
+# What is OXC?
+
+TBD
+
+## Core Team
+
+<VPTeamMembers size="medium" :members="CORE_TEAM_MEMBERS" />
+
+## Community
+
+<VPTeamMembers size="medium" :members="COMMUNITY_TEAM_MEMBERS" />
```

Comment:
@Boshen 

Of course, only if you are fine.
I could redesign UI to show community members with more small cards.